src/main/java/jalau/at18/azul/Factory.java

```diff
@@ -0,0 +1,25 @@
+package jalau.at18.azul;
+
+import java.util.List;
+
+public class Factory {
+
+    private List<Character> tiles;
```

Comment:
I suggested this to @ferAguilar02  and @DaniMB95 , it could be a good idea to use `Tile` enum that Daniela is adding, instead of `Character`, unless I might be missing something, please let me know if this idea makes sense

Reply:
@J0c4  In the Azul group, fer and rodrigo, they use the character instead of tile, because we work in our first baby steps that every class it's independent of the enum and should run their unit test so when we merge this first pull requests of this first baby steps we are going to refactor to use the enum class properly

Reply:
Yes, I used Character because in my branch I don't have the enum that Daniela is adding, so when we merge I was thinking of using Tile instead of Character